frontend/src/components/Answers.js

```diff
@@ -1,7 +1,29 @@
-export default function AnswerCard () {
+import styled from "styled-components/macro"
+
+export default function Answers({answers}) {
 
     return (
-        <>
-        </>
+        <AnswerList>
+            {answers.map(answer =>
+                <button> key= {answer} {answer} </button>
+            )}
```

Comment:
when answers can be undefined you can add ? to have to prevent an exception
```suggestion
            {answers?.map(answer =>
                <button> key= {answer} {answer} </button>
            )}
```